plugins/azure-devops-backend/api-report.md

```diff
@@ -0,0 +1,33 @@
+## API Report File for "@backstage/plugin-azure-devops-backend"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Build } from 'azure-devops-node-api/interfaces/BuildInterfaces';
+import { Config } from '@backstage/config';
+import express from 'express';
+import { GitRepository } from 'azure-devops-node-api/interfaces/GitInterfaces';
+import { Logger as Logger_2 } from 'winston';
+import { WebApi } from 'azure-devops-node-api';
+
+// Warning: (ae-missing-release-tag) "createRouter" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export function createRouter(options: RouterOptions): Promise<express.Router>;
+
+// Warning: (ae-missing-release-tag) "RouterOptions" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export interface RouterOptions {
+  // Warning: (ae-forgotten-export) The symbol "AzureDevOpsApi" needs to be exported by the entry point index.d.ts
+  //
+  // (undocumented)
+  azureDevOpsApi?: AzureDevOpsApi;
```

Comment:
ideally the `AzureDevOpsApi` type should also be exported and appear in this file (as the warning above mentions), so that consumers of this plugin will know how to form a proper implementation of that router argument

Reply:
I've fixed this up, should I also be fixing the other warnings? Not totally sure what to do with them